src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -0,0 +1,146 @@
+      >
+        {profiles.map((profile) => (
+          <div className="profile-item" key={profile.profileId}>
+            <SqSocialConnect profiles={mapProfilesToSocialConnect([profile])} />
+            <span>@{profile.username}</span>
+            <div className="wrapper-selectors toggle">
+              <input
+                type="checkbox"
+                name={`toggle-${profile.username}`}
+                id={`toggle-${profile.profileId}`}
+                checked={profile.hasCreatorsInsights}
+                onChange={() => onToggle && onToggle(profile.profileId, profile.socialNetwork, profile.hasCreatorsInsights)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)